### PR TITLE
Parallel

### PR DESCRIPTION
**Pull request by brmatthy**
# Parallel coordinate plot
image

The lines are highlighted on hover.

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -80,6 +82,20 @@ const gentSVG = cityMap(geoData)
 svg`${gentSVG}`
 ```
 
+## Rankings
+```js
+const years = getYears().map(String);
+const categories = getCategories();
+const year = view(Inputs.select(years, {value: "2018", label: "Selecteer jaar:"}));
```
Comment (matsvbelle) [inline docs/index.md:89]: Maybe add a default value (null) to visualize the results for every year?

File: docs/index.md [hunk 2]
```diff
@@ -80,6 +82,20 @@ const gentSVG = cityMap(geoData)
...
+const category = view(Inputs.select(categories, {value: "Graffiti", label: "Selecteer categorie:"}));
```
Comment (matsvbelle) [inline docs/index.md:90]: Same as above, maybe add a default value to see the results for all crimes at once?

**Review by matsvbelle — APPROVED** [review]
Looks good! I would just add a small change (see comments) that's easy to implement. You can look at my branch (lineChart) where I already did something very similar.

**Comment by brmatthy** [comment]
I'll fix with a second PR

**Pull request merged**